components/builder-depot/src/error.rs

```diff
@@ -85,6 +87,7 @@ impl error::Error for Error {
         match *self {
             Error::BadPort(_) => "Received an invalid port or a number outside of the valid range.",
             Error::DataStore(ref err) => err.description(),
+            Error::Generic(ref err) => err,
```

Comment:
This is a good example - if these weren't generic, this string could describe the error.